doc/concepts/configuration.rst

```diff
@@ -0,0 +1,617 @@
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301,127.0.0.1:3303"
+
+*   Enables :ref:`traffic encryption <enterprise-iproto-encryption>` for a connection using corresponding URI parameters:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "127.0.0.1:3301?transport=ssl&ssl_key_file=localhost.key&ssl_cert_file=localhost.crt&ssl_ca_file=ca.crt"
+
+    Note that traffic encryption is supported by the `Enterprise Edition <https://www.tarantool.io/compare/>`_ only.
+
+
+*   Use a Unix domain socket:
+
+    .. code-block:: yaml
+
+        iproto:
+          listen: "unix/:./{{ instance_name }}.iproto"
+
+
+.. _configuration_options_access_control:
+
+Access control
+~~~~~~~~~~~~~~
+
+The ``credentials`` section allows you to grant the specified privileges to users.
+In the example below, there are two users:
+
+*   The *replicator* user is used for replication and has a corresponding role.
+*   The *client* user has the ``super`` role and can perform any action on Tarantool instances.
+
+..  literalinclude:: /code_snippets/snippets/replication/instances.enabled/manual_leader/config.yaml
+    :language: yaml
+    :lines: 1-8
+    :dedent:
+
+To learn more, see the :ref:`Access control <authentication>` section.
+
+
+.. _configuration_options_memory:
+
+Memory
+~~~~~~
+
+The ``memtx.memory`` option specifies how much memory Tarantool allocates to actually store data.
+
+.. code-block:: yaml
+
+    memtx:
+      memory: 100000000
+
+When the limit is reached, ``INSERT`` or ``UPDATE`` requests fail with :ref:`ER_MEMORY_ISSUE <admin-troubleshoot-memory-issues>`.
+
+
+.. _configuration_options_directories:
+
+Snapshots and write-ahead logs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``snapshot.dir`` and ``wal.dir`` options can be used to configure directories for storing snapshots and write-ahead logs.
+
+.. code-block:: yaml
+
+    instance001:
+      snapshot:
+        dir: 'var/snapshots'
+      wal:
+        dir: 'var/wals'
+
+
+
+.. _configuration_run_instance:
+
+Starting Tarantool instances
+----------------------------
+
+.. _configuration_run_instance_tt:
+
+Starting instances using the tt utility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The :ref:`tt <tt-cli>` utility is the recommended way to start Tarantool instances.
+
+Instance files or directories are organized into applications in the ``instances_enabled`` directory.
+The example below shows how a :ref:`layout <admin-start_stop_instance-multi-instance-layout>` of the application called ``app`` might look:
+
+.. code-block:: none
+
+    instances.enabled
+    └── app
+        ├── config.yaml
+        ├── myapp.lua
+        └── instances.yml
+
+*   ``config.yaml`` is a :ref:`configuration file <configuration_file>`.
+*   ``myapp.lua`` is a Lua script containing an :ref:`application to load <configuration_application>`.
+*   ``instances.yml`` specifies :ref:`instances <admin-start_stop_instance-multi-instance>` to run in the current environment.
```

Comment:
Shouldn't we also place `tt.yaml` here to give a minimal working example?

When I played with it the last time, `tt: {}` in `tt.yaml` was enough.

Reply:
Sure, my bad. Actually, the included configs and samples are placed in the `config` directory with a corresponding  `tt.yaml` file, so a user can play with different settings and features. I'm going to add READMEs for all environments used inside docs.
<img width="246" alt="image" src="https://github.com/tarantool/doc/assets/38073144/5661de87-7d21-4b2e-9e23-86576e7ed063">